src/main/io/osd.c

```diff
@@ -825,6 +825,12 @@ static bool osdDrawSingleElement(uint8_t item)
                 break;
             }
 
+            // Warn when in constant idle mode
+            if (osdWarnGetState(OSD_WARNING_CONSTANT_IDLE) && isConstantIdleMode()) {
+                osdFormatMessage(buff, OSD_FORMAT_MESSAGE_BUFFER_SIZE, "CONSTANT IDLE");
```

Comment:
The buffer is only sized to allow 11 characters for the warning text.  This string is 13 characters which will cause an overflow and step on memory.  You'll need to abbreviate the message.

Reply:
fixed